src/PostList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Post from './Post';
+
+const PostList = ({ posts }) => (
+  posts.map(post => (
+    <Post post={post} />
```

Comment:
Три слова post подряд не гуд для понимания кода
```suggestion
    <Post post={postData} />
```